readme.md

```diff
@@ -106,18 +106,28 @@ When merging into a newer MPS version, follow these steps.
 The example assumes we merge `mps2021.1` into `mps2021.2`.
 
 1. Open the _target_ MPS version (2021.2) on the _target_ branch (`mps2021.2`).
-2. Merge the _source_ branch (`mps2021.1`) into your _current_ branch (`mps2021.2`).
-3. Double-check `gradle.properties` still contains the proper entries for:
+2. Checkout a new branch off of the _target_ branch (`mps2021.2`), named `mps2021.2-migration` (or a name that's as least as good).
+3. Merge the _source_ branch (`mps2021.1`) into your _current_ branch (`mps2021.2-migration`).
+4. Double-check `gradle.properties` still contains the proper entries for:
    * `mpsVersionSuffix` should be the _target_ MPS version (`2021.2`)
    * `mpsVersion` full _target_ MPS version (`2021.2.6`)
    * `mpsExtensionsVersion` latest version of [MPS-extensions](https://jetbrains.github.io/MPS-extensions/) for the _target_ MPS version (`2021.2.2631.1360a64`)
-4. Run the Migration Assistant.
-5. Run all tests.
-6. Update the build model.
-7. Repeat steps 3-6 for the test projects, residing in `test-project/` and `test-project-externalLib/`.
-  **Note** Before opening these projects in MPS:
-    * Make sure to rename their build model files (the `.mps` files in the `models/` directory under the one solution under `solutions/` of either project) to not have the `-ignore` suffix.
-    * Make sure to put those suffixes back in place before committing.
+5. Run the Migration Assistant.
+6. Update the (two) build models, triggering the “Reload Modules From Disk” intention when and where needed.
+7. Check the entire project using “Check Project”.
```

Comment:
Neither _check_, _build_, nor _run all tests_ will work on the whole project:

* Stuff in `xx_broken` is just that.
* Stuff in `io.lionweb.mps.converter.test.mpsextensions` depends on https://github.com/specificlanguages/mps-gradle-plugin/issues/9
* Stuff in `io.lionweb.mps.converter.test.usebroken` depends on `xx_broken`
* Stuff in `io.lionweb.mps.server.test` depends on this MPS being the one listening to the standard MPS port (i.e. no other MPS running)